src/git.rs

```diff
@@ -267,6 +267,8 @@ pub fn add_crate(env: &Environment, krate: Crate) -> Result<(), PerformError> {
     serde_json::to_writer(&mut file, &krate)?;
     file.write_all(b"\n")?;
 
+    println!("Updating crate `{}#{}`", krate.name, krate.vers);
```

Comment:
Let's use `format!` here to get a `String`, and then use that to print and in the call to `commit_and_push` below.